crates/api_decl/src/namespaces/anvil_zks.rs

```diff
@@ -51,6 +51,6 @@ pub trait AnvilZksNamespace {
     ///
     /// # Returns
     /// Bytes with the witness that can be passed to proving system.
-    #[method(name = "getBoojumWitness")]
-    async fn get_boojum_witness(&self, batch_number: L1BatchNumber) -> RpcResult<Bytes>;
+    #[method(name = "getZKsyncOSWitness")]
+    async fn get_zksync_os_witness(&self, batch_number: L1BatchNumber) -> RpcResult<Bytes>;
```

Comment:
Can we call this `getWitness` instead? And make it return an error when invoked in non-zksync-os mode? Technically speaking Era batches also have witness we just never had a need to generate them in anvil-zksync.